.env

```diff
@@ -0,0 +1 @@
+AUTH_SECRET=foKQBSrkRtQE+s555NG/J2HiajswRKi+7vnQEnUUQ6E=
```

Comment:
My mistake, should have included `.env` in `.gitingore`.
**NEVER** make a sensitive credential part of checked code base. I will use a different credential for my project, such that credential is obsolete.